### PR TITLE
Modify search placeholder

### PR DESCRIPTION
**Pull request by uzmannazari**
## Summary
Modification of pages search place holder that can be read from language resource files.

### CODE REVIEW

**Review by mitchelsellers — CHANGES_REQUESTED** [review]
Similar question to the other PR if you are moving this to be pulled from localization we need to update the localization files to include this element.

**Comment by uzmannazari** [comment]
> Similar question to the other PR if you are moving this to be pulled from localization we need to update the localization files to include this element.

The element is already available in the language resource file

**Review by valadas — APPROVED** [review]
Yep appears to be here https://github.com/dnnsoftware/Dnn.Platform/blob/935540d3d05905113532586fe7285e74abbfe137/Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.Pages/App_LocalResources/Pages.resx#L177

Looks good to me

**Review by mitchelsellers — APPROVED** [review]

File: Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ class SearchPageInput extends Component {
                             value={this.state.searchTerm}
                             onChange={this.onSearchFieldChange.bind(this)}
                             onKeyPress={(e) => { e.key === "Enter" ? this.props.onSearch(this.state.searchTerm) : null; }}
-                            placeholder="Search" />
+                            placeholder="{Localization.get("Search")}" />
```
Comment (valadas) [inline Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx:47]: I don't think the double-quotes should be kept here, did you test this out ?

**Review by valadas — CHANGES_REQUESTED** [review]

File: Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx (hunk 1 above)
Comment (uzmannazari) [reply Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx:47]: No, unfortunately, I could not test. I do not have the test conditions.

**Comment by valadas** [comment]
You just have to run ./build.ps1 and get the installer from the artifacts folder and install a site with it. I am pretty sure those quotes need to be removed or it will just literally show that text.

File: Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx (hunk 1 above)
Comment (david-poindexter) [reply Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx:47]: > I don't think the double-quotes should be kept here, did you test this out ?

I agree 100% regarding the outer quotes.

**Comment by david-poindexter** [comment]
The build is failing anyway.  I am pretty sure those outer quotes need to be removed.

File: Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx (hunk 1 above)
Comment (mitchelsellers) [inline Dnn.AdminExperience/ClientSide/Pages.Web/src/components/SearchPage/SearchPageInput.jsx:47]:
```suggestion
                            placeholder={Localization.get("Search")} />
```

**Review by uzmannazari — COMMENTED** [review]
I cant test personabar but i think this will work fine

**Review by mitchelsellers — APPROVED** [review]

**Review by valadas — APPROVED** [review]

**Pull request merged**